app/handlers.py

```diff
@@ -188,7 +187,7 @@ def plaid_bond_test(account_id, PLAID_ENV):
 
       function initializePlaidLink( data ) {{
         const handler = Plaid.create({{
-          env: {PLAID_ENV},
+          env: '{PLAID_ENV}',
```

Comment:
added quotes

Reply:
This will already be there, right?

Reply:
the whole thing is a string, need explicit quotes around this string, else getting:

```
52 Uncaught (in promise) ReferenceError: sandbox is not defined
    at initializePlaidLink (52)
    at 92
```

Reply:
Thanks for confirming